tools/preflight/cleanup.go

```diff
@@ -13,6 +13,8 @@ import (
 
 type CleanupOptions struct {
 	CommonOptions
+	CleanupMode string `json:"cleanupMode"`
```

Comment:
Determines whether the cleanup is to be done for all preflight resources or for a particular preflight uid
if mode is 'all' then clean all resources
if mode is 'uid' then a uid field must be specified in yaml input file.

For CLI, there is no need to specify cleanup-mode. if --uid flag is specified then, mode is uid, all otherwise

Reply:
ok, why can't we keep the same behavior as CLI flag? if `uid` value is empty or not given in yaml file, cleanup all resources. If uid value is given, cleanup that uid resources only.